cdt_slugs.yaml

```diff
@@ -483,6 +483,16 @@ python-javapackages:
   custom: false
   skipped_cdts:
     - cos6-x86_64
+rdma-core:
+  custom: false
+  skipped_cdts:
+    - cos6-x86_64
+  license_file: licenses/rdma-core-licenses
+rdma-core-devel:
+  custom: false
+  skipped_cdts:
+    - cos6-x86_64
+  license_file: licenses/rdma-core-licenses
```

Comment:
So there are multiple license files for `rdma-core`. Have added them in a directory. However it seems like the script doesn't like this. Is there a particular way I should be specifying them?

Reply:
NVM. Saw an example in this file. Have updated accordingly in the latest commit.